Update Understanding Cloudflare HTTP2 and HTTP3 Support.md (#8658)
cloudflare/cloudflare-docs: content/support/Network/Understanding Cloudflare HTTP2 and HTTP3 Support.md

## Overview

> HTTP/2 and HTTP/3 accelerate page load. They are both free for all [Cloudflare plans](http://www.cloudflare.com/plans) but require an [SSL certificate at Cloudflare’s edge network](/ssl/get-started/). Configure HTTP/2 and HTTP/3 via the Cloudflare **Network** app. HTTP/2 is enabled by default and domains on Free plans cannot disable it.

Reviewer: :)
```suggestion
HTTP/2 and HTTP/3 accelerate page load. They are both free for all [Cloudflare plans](https://www.cloudflare.com/plans) but require an [SSL certificate at Cloudflare’s edge network](/ssl/get-started/). Configure HTTP/2 and HTTP/3 via the Cloudflare **Network** app. HTTP/2 is enabled by default and domains on Free plans cannot disable it.
```

> HTTP/2 and HTTP/3 accelerate page load. They are both free for all [Cloudflare plans](http://www.cloudflare.com/plans) but require an [SSL certificate at Cloudflare’s edge network](https://support.cloudflare.com/hc/articles/203295200#h_036e2e20-96d8-4199-bb1f-0fbb41b5cdd0). Configure HTTP/2 and HTTP/3 via the Cloudflare **Network** app. HTTP/2 is enabled by default and domains on Free plans cannot disable it.

Reviewer 2: ```suggestion
HTTP/2 and HTTP/3 accelerate page load. They are both free for all [Cloudflare plans](http://www.cloudflare.com/plans) but require an [SSL certificate at Cloudflare’s edge network](/ssl/get-started/). Configure HTTP/2 and HTTP/3 via the Cloudflare **Network** app. HTTP/2 is enabled by default and domains on Free plans cannot disable it.
```

This KB 301's to the SSL docs here

Author: WFM

> HTTP/2 and HTTP/3 accelerate page load. They are both free for all [Cloudflare plans](http://www.cloudflare.com/plans) but require an [SSL certificate at Cloudflare’s edge network](https://support.cloudflare.com/hc/articles/203295200#h_036e2e20-96d8-4199-bb1f-0fbb41b5cdd0). Configure HTTP/2 and HTTP/3 via the Cloudflare **Network** app. HTTP/2 is enabled by default and requires and domains on Free plans cannot disable it.

Author: ```suggestion
HTTP/2 and HTTP/3 accelerate page load. They are both free for all [Cloudflare plans](http://www.cloudflare.com/plans) but require an [SSL certificate at Cloudflare’s edge network](https://support.cloudflare.com/hc/articles/203295200#h_036e2e20-96d8-4199-bb1f-0fbb41b5cdd0). Configure HTTP/2 and HTTP/3 via the Cloudflare **Network** app. HTTP/2 is enabled by default and domains on Free plans cannot disable it.
```

> A browser and web server automatically negotiate the highest protocol available. Thus, when both HTTP/3 and HTTP/2 are enable, HTTP/3 takes precedence. 

Author: ```suggestion
A browser and web server automatically negotiate the highest protocol available. Thus, when both HTTP/3 and HTTP/2 are enabled, HTTP/3 takes precedence. 
```

-   HTTP header compression - can reduce the number of bytes required to exchange headers, which speeds up requests and responses.

Note:

-   How browsers choose which requests to multiplexing in a single connection is a complext topic. Typically a connection applies to a single domain. However, HTTP/2 supports [connection coalescing](https://www.rfc-editor.org/rfc/rfc9113.html#name-connection-reuse), where clients can carry out additional checks and if they pass, requests for different domains can also be multiplexed.

___

## HTTP/3

HTTP/3 uses QUIC, which is a secure-by-default transport protocol. HTTP/3 improves page load times in a similar way to HTTP/2. However, the QUIC transport protocol solves TCP's head-of-line blocking problem, meaning that performance over lossy networks can be better. Enable HTTP/3 via the Cloudflare **Network** app. 

For more information, refer to the [Learning center](https://www.cloudflare.com/learning/performance/what-is-http3/).

___

## Server Push

> The Server Push feature allows origin web servers to send resources to the client or web browser without waiting to for the client to discover they are needed. In practice, this protocol feature was hard to leverage and could sometimes make page load times worse. [Early Hints](https://developers.cloudflare.com/pages/platform/early-hints/#early-hints) has emerged as a replacement solution that avoids some of the pitfalls of Server Push.

Reviewer 3: ```suggestion
The Server Push feature allows origin web servers to send resources to the client or web browser without waiting to for the client to discover they are needed. In practice, this protocol feature was hard to leverage and could sometimes make page load times worse. [Early Hints](/cache/about/early-hints/) has emerged as a replacement solution that avoids some of the pitfalls of Server Push.
```

> The Server Push feature allows origin web servers to send resources to the client or web browser without waiting to for the client to discover they are needed. In practice, this protocol feature was hard to leverage and could sometimes make page load times worse. [Early Hints](https://developers.cloudflare.com/pages/platform/early-hints/#early-hints) has emerged as a replacement solution that avoids some of the pitfalls of Server Push.

Reviewer 2: Is it worth linking to the [generic Early Hints](https://developers.cloudflare.com/cache/about/early-hints/) page instead? The Pages one is specific to how it works (and disabling it) on Pages whereas the cache one applies to anything proxied on a user's zone.

Author: Sure!
